### PR TITLE
Don't tick a group if its player is in the pending state.

### PR DESCRIPTION
**Pull request by rjwright**
This addresses https://github.com/web-animations/web-animations-next/issues/267

### CODE REVIEW

**Comment by rjwright** [comment]
@shans @dstockwell

File: src/group-constructors.js [hunk 1]
```diff
@@ -55,6 +55,8 @@
     var underlyingPlayer;
     var ticker = function(tf) {
       var player = underlyingPlayer._wrapper;
+      if (player.playState == 'pending') return;
```
Comment (dstockwell) [inline src/group-constructors.js]: In https://github.com/web-animations/web-animations-next/blob/master/src/effect-callback.js#L34 we probably want to also call the effect callback when the playState changes.

**Comment by dstockwell** [comment]
Test?

File: src/group-constructors.js (hunk 1 above)
Comment (shans) [inline src/group-constructors.js]: Maybe! It's not relevant to this bug fix though, is it?

Comment (dstockwell) [inline src/group-constructors.js]: I'm not sure. Seems like there could be cases where we never get another call now, but I didn't look in detail.

Comment (shans) [inline src/group-constructors.js]: I think we need to determine whether the specification should change to permit us calling again first. Let's commit this for now and work out the specification question later.

**Comment by shans** [comment]
lgtm

**Pull request merged**